### PR TITLE
LPS-186200 - Objects sidebar has the wrong padding, background and and has styling in the Tabs component

### PR DESCRIPTION
**Pull request by Artartur**
Related Issues: [LPS-186200](https://issues.liferay.com/browse/LPS-186200)

### CODE REVIEW

**Comment by Artartur** [comment]
ci:test:relevant

**Comment by Artartur** [comment]
ci:test:object

**Comment by alinedoleron** [comment]
Just started reviewing :)

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -2,13 +2,8 @@
...
 
-.side-panel-iframe__tabs {
-	background-color: $white;
-	padding: 0.5rem 1rem 0;
-
-	.nav-link {
-		padding: 1rem;
-	}
+.nav-tabs + .tab-content .tab-pane {
+	padding: 1.5rem;
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/css/main.scss:6]: Hi @Artartur. I've tested the PR and I noticed the following requirement ([ticket](https://issues.liferay.com/browse/LPS-186200)) was not being applied correctly for all the panels.

> The padding between the card and the sidebar should be 24px.

I talked to Julia and she said we need to get the same look of Field's sidepanel that is on master. So I made some tests now and I think the correct padding should be `0rem`.

image

Please, could you test if this fixes all the panels?
```
.lfr-objects__side-panel-content {
	.nav-tabs + .tab-content .tab-pane {
		padding: 0rem;
	}
}
```

Thanks! :relaxed:

**Review by alinedoleron — CHANGES_REQUESTED** [review]
Hi @Artartur . Please, check my comment above. Thanks! =D

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/css/main.scss]: Yeah you're right, I've tested here your request with the code you sent and it fixed all the panels. Thx @alinedoleron

Comment (alinedoleron) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/css/main.scss]: Thank you for the quick response and update, @Artartur! ☺️

**Comment by Artartur** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (Artartur) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/css/main.scss]: My bad, I fixed the SF problem

**Comment by Artartur** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:object

**Review by alinedoleron — APPROVED** [review]

**Comment by alinedoleron** [comment]
ci:forward:force

**Pull request closed without merging**